src/gooddata.ts

```diff
@@ -39,6 +40,7 @@ export class SDK {
     public execution: ExecutionModule;
     public project: ProjectModule;
     public report: ReportModule;
+    public dashboard: DashboardModule;
```

Comment:
I would expect here ExportModule which is providing just exportDashboardToPdf function

Reply:
right know its mix of two approaches
is export module which provides functionality to export dashboard to pdf ?
or
is dashboard  module which provides functionality to export to pdf?

choose just one :) not both :) and align folder, module names accordingly :)

Reply:
New `DashboardExportModule` is more reasonable now :)

Reply:
please comment below... with my proposal

Reply:
Fixed